components/develop/modules/ROOT/nav.adoc

```diff
@@ -37,6 +37,7 @@
 ** xref:ml.adoc[Using SKALE-ML]
 ** xref:ima::index.adoc[Performing IMA Bridge transfers]
 ** xref:metaport::index.adoc[Using Metaport Widget]
+** xref:using-metaport-bridge.adoc[Using Metaport Bridge]
```

Comment:
Suggestion Only, Not mandatory change - Flip these so that Bridge comes before Widget